INSTRUCTION.md

```diff
@@ -0,0 +1,26 @@
+# How to Validate the Application Setup
+
+## Run the bootstrap script:
+
+```
+./bootstrap.sh
+```
+
+## Verify Resources
+Ensure all resources are succesfully created using command:
+```
+kubectl get all -n todoapp
+```
+## Check status of pods:
+```
+kubectl get pods -n todoapp
+kubectl get pv -n todoapp
+kubectl get pvc -n todoapp
+```
+
+## Connect to pods
+```
+kubectl exec "pod name" -n todoapp -it --sh
```

Comment:
The command 'kubectl exec' should use '-- /bin/sh' or '-- /bin/bash' instead of '--sh' to specify the shell to use. The correct command should be:
kubectl exec "pod name" -n todoapp -it -- /bin/sh